infra/_modules/cms_function_app/networking.tf

```diff
@@ -0,0 +1,20 @@
+resource "azurerm_private_endpoint" "st_queuest_queue" {
+  name                = "${var.prefix}-${var.env_short}-${var.location_short}-${var.domain}-queuest-queue-pep-01"
```

Comment:
**chore:** use `cms` instead of `queue` to indicate this private endpoint is related to `cms` storage account
```suggestion
  name                = "${var.prefix}-${var.env_short}-${var.location_short}-${var.domain}-cms-st-queue-pep-01"
```